### ISSUE TITLE
Debug-Messages on .net DefaultTraceListener

### ISSUE THREAD

**Issue by tajbender** [SunburstApps/Sunburst.WindowsForms.Ribbon#2]
Hi William,

currently I'm using a `DefaultTraceListener` for debugging and logging purposes in my Application. Using your Ribbon it gets filled with `UpdateProperty key` and `Execute verb`-Messages.

This was already the case while I used the original components by Arik Poznanski and Bernhard Elbl, aka `Ribbon.dll`. It's not a big problem, but perhaps you could take a look at it.

Most of these messages are generated when starting the Application, however, some get fired later. The `Debug: 00:44:47`- and `Scope: 00:44:46`-Messages are my own, just for your Knowledge.

```
> NEW SESSION STARTED...

Scope: 00:44:46 Enter Scope of Member .ctor @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\AppContext.cs
Scope: 00:44:46 Enter Scope of Member .ctor @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs
Debug: 00:44:46 AddClipboardFormatListener @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in .ctor
Debug: 00:44:46 Activating '' @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in ActiveShellBrowserChanged
Debug: 00:44:46 DpnDockPanel_ActiveContentChanged, Type:WeifenLuo.WinFormsUI.Docking.DockPanel @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in DpnDockPanel_ActiveContentChanged
UpdateProperty key: Enabled for command 19903
UpdateProperty key: ItemsSource for command 19901
UpdateProperty key: Label for command 19999
UpdateProperty key: Enabled for command 1101
UpdateProperty key: Enabled for command 1102
UpdateProperty key: Enabled for command 1103
UpdateProperty key: Enabled for command 1201
UpdateProperty key: Enabled for command 1202
UpdateProperty key: Enabled for command 1203
UpdateProperty key: LargeImage for command 1100
UpdateProperty key: SmallImage for command 1100
UpdateProperty key: LargeImage for command 1200
UpdateProperty key: SmallImage for command 1200
Debug: 00:44:47 Activating '' @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in ActiveShellBrowserChanged
Debug: 00:44:48 Firing of ItemsChanged event. @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Components\DockContents\ShellBrowserDockContent.cs' in UIDecouplingTimer_Tick
Debug: 00:44:48 Firing of SelectionChanged event. @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Components\DockContents\ShellBrowserDockContent.cs' in UIDecouplingTimer_Tick
Debug: 00:44:48 Firing of ItemsChanged event. @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Components\DockContents\ShellBrowserDockContent.cs' in UIDecouplingTimer_Tick
UpdateProperty key: Enabled for command 100
UpdateProperty key: Enabled for command 101
UpdateProperty key: Enabled for command 102
UpdateProperty key: Enabled for command 103
UpdateProperty key: Enabled for command 110
UpdateProperty key: Enabled for command 111
UpdateProperty key: Enabled for command 120
UpdateProperty key: Enabled for command 121
UpdateProperty key: Enabled for command 125
UpdateProperty key: Enabled for command 130
Debug: 00:44:56 DpnDockPanel_ActiveContentChanged, Type:WeifenLuo.WinFormsUI.Docking.DockPanel @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in DpnDockPanel_ActiveContentChanged
Debug: 00:45:01 Activating '[Download]' @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in ActiveShellBrowserChanged
Execute verb: Execute for command 101
Scope: 00:45:01 Enter Scope of Member CmdAppOpenNewShellBrowserPane_Execute @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs
Scope: 00:45:01 Enter Scope of Member CreateNewFileBrowser @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs
Debug: 00:45:01 Activating '' @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in ActiveShellBrowserChanged
Debug: 00:45:01 Firing of ItemsChanged event. @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Components\DockContents\ShellBrowserDockContent.cs' in UIDecouplingTimer_Tick
Debug: 00:45:01 Firing of SelectionChanged event. @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Components\DockContents\ShellBrowserDockContent.cs' in UIDecouplingTimer_Tick
Debug: 00:45:10 DpnDockPanel_ActiveContentChanged, Type:WeifenLuo.WinFormsUI.Docking.DockPanel @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in DpnDockPanel_ActiveContentChanged
Debug: 00:45:12 Activating 'Desktop' @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in ActiveShellBrowserChanged
Debug: 00:45:13 DpnDockPanel_ActiveContentChanged, Type:WeifenLuo.WinFormsUI.Docking.DockPanel @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in DpnDockPanel_ActiveContentChanged
Debug: 00:45:18 Activating 'Desktop' @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in ActiveShellBrowserChanged
UpdateProperty key: TooltipTitle for command 19999
UpdateProperty key: TooltipDescription for command 19999
UpdateProperty key: TooltipDescription for command 19999
Debug: 00:45:49 RemoveClipboardFormatListener @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in Electrifier_FormClosed
Debug: 00:45:49 DpnDockPanel_ActiveContentChanged, Type:WeifenLuo.WinFormsUI.Docking.DockPanel @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\Forms\Electrifier.cs' in DpnDockPanel_ActiveContentChanged
Scope: 00:45:49 Enter Scope of Member AppContext_ThreadExit @ 'S:\[Git.Workspace]\electrifier\src\electrifier.Core\AppContext.cs
```

If you need the source, it can be found here: [https://github.com/electrifier/electrifier](https://github.com/electrifier/electrifier). If you need running binaries, just leave a message.

Thanks in advance and regards,
tajbender

**Comment by wjk** [comment]
Should be fixed with version 2.1.0 of the NuGet package. Sorry for the delay; for some reason, I never saw the ping that you opened this issue. Thanks!

**Comment by tajbender** [comment]
Looks good! Thank you very much!
